app/build.gradle

```diff
@@ -1,11 +1,31 @@
+def getAPIKey(String label)
+{
+    def secretsFile = file("tokens.dat")
+    try
+    {
+        def val = secretsFile.text.lines().find(s -> s.contains(label)).split(':')[1].trim()
+        println("Reading key " + label + " from file")
+        return val
+    }
+    catch(FileNotFoundException ignored)
```

Comment:
This case is for the CI?
You should maybe add an case if the file is not found, and the env does not contain the info
(the `broken` case)

Reply:
Will do